packages/client/lib/commands/PUBSUB_SHARDNUMSUB.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { strict as assert } from 'assert';
+import testUtils, { GLOBAL } from '../test-utils';
+import { transformArguments } from './PUBSUB_SHARDNUMSUB';
+
+describe('PUBSUB SHARDNUMSUB', () => {
+    describe('transformArguments', () => {
+        it('simple', () => {
+            assert.deepEqual(
+                transformArguments(),
+                ['PUBSUB', 'SHARDNUMSUB']
+            );
+        });
+
+        it('string', () => {
+            assert.deepEqual(
+                transformArguments('channel'),
+                ['PUBSUB', 'SHARDNUMSUB', 'channel']
+            );
+        });
+
+        it('array', () => {
+            assert.deepEqual(
+                transformArguments(['1', '2']),
+                ['PUBSUB', 'SHARDNUMSUB', '1', '2']
+            );
+        });
+    });
+
+    testUtils.testWithClient('client.pubSubShardNumSub', async client => {
+        assert.deepEqual(
```

Comment:
can you please make the test "non empty" so it'll actually test `transforReply` as well?